scvi/data/_preprocessing.py

```diff
@@ -452,3 +452,36 @@ def add_dna_sequence(
     sequence_df = pd.concat(seq_dfs, axis=0).loc[adata.var_names]
     adata.varm[sequence_varm_key] = sequence_df
     adata.varm[code_varm_key] = sequence_df.applymap(_dna_to_code)
+
+
+def reads_to_fragments(
+    adata: anndata.AnnData, layer: Optional[str] = None, copy: bool = False
+) -> Optional[anndata.AnnData]:
+    """Convert read counts to appoximate fragment counts.
+
+    Parameters
+    ----------
+    adata
+        AnnData object that contains read counts.
+    layer
```

Comment:
I think this function should either:

1. just return the new scipy sparse matrix based on the input layer, then the user can put it wherever they want and we could remove the copy option.

or 

2. Have a read_layer argument for the input and a `fragment_layer` argument for the output layer, which should default to `fragments`

Reply:
I opted for the second option now.